package.json

```diff
@@ -2,5 +2,12 @@
   "private": true,
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "scripts": {
+    "precommit": "lerna run --concurrency 1 --stream precommit"
+  },
+  "devDependencies": {
+    "lerna": "2.4.0",
+    "husky": "^0.13.3"        
```

Comment:
Heads up, I've fixed this in this PR: https://github.com/jerelmiller/redux-simple-auth/pull/41